client/my-sites/checkout/utils.ts

```diff
@@ -135,3 +144,49 @@ export function getAffiliateCouponLabel(): string {
 	// translators: The label of the coupon line item in checkout
 	return translate( 'Exclusive Offer Applied' );
 }
+
+export function useGetWpcomPlanTotalIfPaidMonthly( products: Array< ResponseCartProduct > ): {
```

Comment:
I think best to not name hooks like `useGetFoo`. Just name is as you would the selector `useFoo`, so `useWpcomPlanTotalIfPaidMonthly`.

Although something strikes me as very odd reading the name here. It's complicated, as is the code underneath.

Can you add a description?

Reply:
Done.